int64 robotRow
int64 robotCol
---
bool clash
